css/style.css

```diff
@@ -1,19 +1,157 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
+}
+.navbar .logo {
+  height: 60px;
+  margin-left: 30px;
+  float: left;
+}
+.navbar a{
+  float: left;
+  margin-top: 0;
+  display: block;
+  color: grey;
+  text-align: center;
+  padding: 14px 20px;
+  background-color: white;
+}
+.navbar a:hover{
+  color: orange;
+}
+.right {
+  float: right;
+}
+a{
+  text-decoration: none;
+}
+.navbar a.current-tab{
+  color: black;
+}
+.navbar .icon {
+  display: none;
+}
+
+
+/*Introduction General styles*/
+.intro{
+  background-image: url(/img/first-background.jpg);
+  background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center bottom;
+  width: 100%;
+  height: auto;
+  overflow: auto;
+}
+  
+.intro .content{
+  color: white;
+  text-align: center;
+  padding: 100px;
+  padding-top: 0px;
+}
+.intro h1{
+  font-size: 50px;
+  margin-top: 200px;
+  font-weight: 100;
+}
+.intro h3{
+  font-size: 20px;
+  margin-top: 20px;
+  font-weight: 100;
+}
+
+/*Button - general styles*/
+.button{
+  background-color: orange;
+  color: white;
```

Comment:
To follow the design a little more closely, consider adding padding inside the button, as well as giving it a border-radius.

Reply:
Updated CSS to meet this